db/fixtures/miq_alerts.yml

```diff
@@ -321,3 +321,45 @@
             value: prod
   :db: Vm
   :guid: 89db0be8-c240-11de-a3be-000c290de4f9
+- description: Middleware Heap Max > 90%
+  options:
+    :notifications:
+      :delay_next_evaluation: 600
+      :evm_event: {}
+  db: MiddlewareServer
+  responds_to_events: hawkular_alert
+  hash_expression:
+    :eval_method: mw_heap_used
+    :mode: internal
+    :options:
+      :value_mw_greater_than: '90'
+      :value_mw_less_than: '0'
```

Comment:
if the condition is named > 90% in the description, why does it include the less_than 0 part ?

Reply:
I guess the UI is placing the restriction, since the user must specify both values to be able to create the alert. And the provider is coded following this assumption. 
AFAIK, the valid range of this metric is 0% to 100%. So, the "lest_than 0%" condition should never happen, effectively eliminating the lower bound.

Reply:
For GA I would suggest that we actually remove this multiple threshold approach for %heapUsed and just make it a single threshold.  I don't think there is a lot of need for alerting on unused heap, but if so, it could always be done using a second alert.  It would simplify the UI to a basic threshold, avoid this sort of dummy condition definition, and avoid an alert that really is looking for two different things.

Reply:
The less_than 0 part seems odd to me. @h-kataria Does UI place the restriction? Is it necessary?